### ISSUE TITLE
Nebula - AttributeError: NoneType object has no attribute startswith

### ISSUE THREAD

**Issue by matthuisman** [matthuisman/slyguy.addons#451]
REDDIT

Hey, I just re-subscribed to Nebula and unlike the last time I used the plugin, it's not working... Everything is up to date, I'm running Libreelec and re-installing didn't help either. When I start a video (or try to choose the quality) I get AttributeError: NoneType object has no attribute startswith (0lus a lot of python lines :)

**Comment by carlreinke** [comment]
I'm also hitting this issue.
```
error <general>: slyguy.nebula - 'NoneType' object has no attribute 'startswith'
  Traceback (most recent call last):
    File "/home/xbmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
      yield
    File "/home/xbmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
      function(**params)
    File "/home/xbmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
      item = f(*args, **kwargs)
    File "/home/xbmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 53, in decorated_function
      return f(*args, **kwargs)
    File "/home/xbmc/.kodi/addons/slyguy.nebula/resources/lib/plugin.py", line 402, in play
      url, subtitles = api.play(slug)
    File "/home/xbmc/.kodi/addons/slyguy.nebula/resources/lib/api.py", line 220, in play
      resp = self._session.head(data['iframe'])
    File "/usr/lib/python3/dist-packages/requests/sessions.py", line 577, in head
      return self.request('HEAD', url, **kwargs)
    File "/home/xbmc/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 347, in request
      if not url.startswith('http'):
  AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by matthuisman** [comment]
should now be fixed with v0.2.3 just pushed
